draft-ietf-httpbis-cache-latest.xml

```diff
@@ -264,7 +264,8 @@
 </t>
 <iref item="cache key" />
 <t>
-   The <x:dfn>cache key</x:dfn> is comprised of, at a minimum, the request method and target
+   The <x:dfn>cache key</x:dfn> is the information used to select a response and
```

Comment:
```suggestion
   The <x:dfn>cache key</x:dfn> is the information a cache uses to select a response and
```
Maybe avoid the passive voice in this case.

Reply:
But it's my friend.... 

I'm not thrilled about the repeated "cache... cache" but yes this is better.